libs/blocks/global-navigation/global-navigation.js

```diff
@@ -696,7 +702,7 @@ class Gnav {
         this.elements.mainNav.style.removeProperty('padding-bottom');
       } else {
         const offset = Math.ceil(this.elements.topnavWrapper.getBoundingClientRect().bottom);
-        this.elements.mainNav.style.setProperty('padding-bottom', `${offset}px`);
+        this.elements.mainNav.style.setProperty('padding-bottom', `${2 * offset}px`);
```

Comment:
This issue occurs specifically on iPhone devices when there are multiple links in that section and scrolling becomes necessary. I understand that a CSS fix would eliminate the need for extra spacing below. However, the fix we applied is for iPhones using Chrome and Safari browsers:

Case 1: When the URL is loaded and the menu is opened via the hamburger icon, some links become hidden due to the presence of back/forward arrows for navigation in the lower section. See the screenshot for reference:

<img src="https://github.com/user-attachments/assets/756a28e0-9895-4121-89ce-97b4fe1fdf04" width="50%" height="50%">

Case 2: When the page is scrolled slightly, the bottom section disappears, allowing all the links to become visible.

<img src="https://github.com/user-attachments/assets/5174e34c-5c43-42e0-80d9-16053c00952a" width="50%" height="50%">

To resolve Case 1, we increased the padding size. We can move this padding-bottom adjustment to CSS instead of using JavaScript, and remove that functionality from the global navigation.